### PR TITLE
feat: Brochure Creation Notebook

### PR DESCRIPTION
**Pull request by slatawa**
# Description

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

- [ ] Follow the [`CONTRIBUTING` Guide](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/CONTRIBUTING.md).
- [ ] You are listed as the author in your notebook or README file.
  - [ ] Your account is listed in [`CODEOWNERS`](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/.github/CODEOWNERS) for the file(s).
- [ ] Make your Pull Request title in the <https://www.conventionalcommits.org/> specification.
- [ ] Ensure the tests and linter pass (Run `nox -s format` from the repository root to format).
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: language/use-cases/prod-catalog-enrichment/genai_prod_catalog_enrichment.ipynb [hunk 1]
```diff
@@ -50,12 +50,12 @@
...
   {
    "cell_type": "markdown",
-   "id": "2fbcb31c",
+   "id": "45633f2d-d69e-47a3-9ddc-c93fa91caba7",
    "metadata": {},
    "source": [
     "| | |\n",
     "|-|-|\n",
-    "|Author(s) | [Sanchit Latawa](https://github.com/slatawa) |"
+    "|Author(s) | [Sanchit Latawa](https://github.com/slatawa) , Lalit Kumar Digala"
```
Comment (holtskinner) [inline language/use-cases/prod-catalog-enrichment/genai_prod_catalog_enrichment.ipynb:58]: Can you add the other author's github page?

**Review by holtskinner — CHANGES_REQUESTED** [review]
See comments

File: vision/use-cases/broucher-creation-using-imagen/broucher-creation-using-imagen.ipynb
Comment (holtskinner) [inline vision/use-cases/broucher-creation-using-imagen/broucher-creation-using-imagen.ipynb:1]: For the directory and file names, correct spelling of "brochure" (Unless that's a regional spelling difference)

**Comment by slatawa** [comment]
<p>Incorporated </p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055054'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>Changed </p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055074'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>Updated </p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055107'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>Updated</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055173'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>Updated</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055173'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>have removed these now </p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055202'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>Added description </p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055231'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>Added details on this </p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055273'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>this has been removed now </p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055312'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>Updated </p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055336'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>updated </p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055360'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>Updated</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055388'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>Removed</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055467'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>Added</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055521'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>UPdated </p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055582'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>removed</p><p> </p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055614'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>UPDATED</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055646'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>updated</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055682'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>updated</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055766'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>Done</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055132'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>Done</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055441'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>Resolved</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055415'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>Resolved</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055556'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>This is taken as reference color list from Pillow library , we do not have the supported color list.</p><p> </p><p><a href="https://github.com/python-pillow/Pillow/blob/main/src/PIL/ImageColor.py" target="_blank">https://github.com/python-pillow/Pillow/blob/main/src/PIL/ImageColor.py</a>.  (line number 165)</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055705'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by slatawa** [comment]
<p>we want to keep a wrapper for UI handshake and this is the function where any preprocessing will happen.</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/460/discussion/#comment-2023055731'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: language/use-cases/prod-catalog-enrichment/genai_prod_catalog_enrichment.ipynb (hunk 1 above)
Comment (slatawa) [reply language/use-cases/prod-catalog-enrichment/genai_prod_catalog_enrichment.ipynb:58]: Added

**Review by holtskinner — APPROVED** [review]

**Pull request merged**